### PR TITLE
Document comprehensive list of required MATLAB toolboxes

### PR DESCRIPTION
**Pull request by jamiesnape**
From the last million or so license server log entries (which is around a week).

---
This change is [Reviewable](https://reviewable.io/reviews/robotlocomotion/drake/4014)

### CODE REVIEW

**Comment by jamiesnape** [comment]
+@david-german-tri for feature and platform review.

**Comment by david-german-tri** [comment]
> From the last million or so license server log entries (which is around a week).

:laughing:

**Comment by david-german-tri** [comment]
:lgtm:, but ugh

---

Review status: 0 of 1 files reviewed at latest revision, all discussions resolved, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/robotlocomotion/drake/4014#-:-KVfDUP1V8AwuJ-1OXTY:b-iy80by)*

**Comment by jamiesnape** [comment]
@drake-jenkins-bot retest this please

File: drake/doc/installation.rst [hunk 1]
```diff
@@ -27,11 +27,15 @@ Drake is transitioning from being primarily a MATLAB-based toolbox to a C++ tool
 
 Many of the features also use
 
-* Simulink
-* Simulink 3D Animation Toolbox
-* Control Systems Toolbox
+* Control System Toolbox
 * Curve Fitting Toolbox
-* Optimization Toolbox (as a fallback alternative to commercial solvers)
+* Image Processing Toolbox
```
Comment (RussTedrake) [inline drake/doc/installation.rst]: this would be super easy to purge.  it is a false dependency.

File: drake/doc/installation.rst [hunk 2]
```diff
@@ -27,11 +27,15 @@ Drake is transitioning from being primarily a MATLAB-based toolbox to a C++ tool
...
+* Optimization Toolbox
+* Simulink
+* Simulink 3D Animation
```
Comment (RussTedrake) [inline drake/doc/installation.rst]: this is only used as a fallback if the director doesn't launch.

File: drake/doc/installation.rst [hunk 3]
```diff
@@ -27,11 +27,15 @@ Drake is transitioning from being primarily a MATLAB-based toolbox to a C++ tool
...
+* Statistics and Machine Learning Toolbox
```
Comment (RussTedrake) [inline drake/doc/installation.rst]: would expect this is minimal (and removable) at best.

File: drake/doc/installation.rst [hunk 4]
```diff
@@ -27,11 +27,15 @@ Drake is transitioning from being primarily a MATLAB-based toolbox to a C++ tool
...
+* System Identification Toolbox
```
Comment (RussTedrake) [inline drake/doc/installation.rst]: probably real, but only for a (very) few examples.

**Comment by jamiesnape** [comment]
I will create some tickets to purge those toolboxes, if possible.

File: drake/doc/installation.rst (hunk 3 above)
Comment (jamiesnape) [inline drake/doc/installation.rst]: #4020

File: drake/doc/installation.rst (hunk 1 above)
Comment (jamiesnape) [inline drake/doc/installation.rst]: #4019

**Pull request merged**